test/resources/sampleContainer/example1.ttl

```diff
@@ -2,6 +2,8 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/> .
 @prefix ex: <http://example.org/stuff/1.0/> .
 
+</sampleContainer/example*> a :glob.
```

Comment:
@deiu I've replaced the `<example*> a :glob` statement with the example you & @dan-f suggested.